editor/src/components/canvas/canvas-strategies/absolute-resize-bounding-box-strategy.tsx

```diff
@@ -46,9 +47,10 @@ export const absoluteResizeBoundingBoxStrategy: CanvasStrategy = {
     ) {
       return canvasState.selectedElements.every((element) => {
         const elementMetadata = MetadataUtils.findElementByElementPath(metadata, element)
+        const elementProps = allElementProps[EP.toString(element)] ?? {}
         return (
           elementMetadata?.specialSizeMeasurements.position === 'absolute' &&
-          hasAtLeastTwoPinsPerSide(elementMetadata.props)
+          hasAtLeastTwoPinsPerSide(elementProps)
```

Comment:
I think it's worth adding as a comment – I remember that using the spied props was a path of least resistance here, but I _think_ using the actual JSXElement would be more reliable, since we want to understand what are the props we can actually find in code so we can actually change them. 

```suggestion
          hasAtLeastTwoPinsPerSide(elementProps) // TODO should this use projectContents?
```